_docs/1-demo/bookinfo.md

```diff
@@ -182,7 +184,8 @@ from your browser and you should see the bookinfo application `productpage`
 displayed.  Notice that the `productpage` is displayed, with no rating
 stars since `reviews:v1` does not access the ratings service.
 
-**Note**: Replace `GATEWAY_HOST_PORT` with the output of the following command:
+**Note**: Replace `GATEWAY_URL` with the output of the following command:
```

Comment:
Do you think it would make sense to rename the  `GATEWAY_HOST_PORT` environment variable to `GATEWAY_URL`? `_HOST_PORT` makes it sound like it contains a port, not a URL. It would also make this step clearer.